pkg/controller/profilebundle/profilebundle_controller.go

```diff
@@ -490,6 +490,7 @@ func (r *ReconcileProfileBundle) newWorkloadForBundle(pb *compliancev1alpha1.Pro
 							},
 							Env: []corev1.EnvVar{
 								corev1.EnvVar{Name: "PLATFORM", Value: utils.GetPlatform()},
+								corev1.EnvVar{Name: "CONTROL_PLANE_TOPOLOGY", Value: utils.GetControlPlaneTopology()},
```

Comment:
We need this because platform is no longer sufficient to denote ROSA environments from hypershift?

Reply:
this is needed to distinguish ROSA HCP from classic ROSA

Reply:
Ah - interesting. Thanks!